src/views/RejectTransfer.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react';
+import cn from 'classnames';
+import { Grid, Text } from 'indigo-react';
+import * as azimuth from 'azimuth-js';
+
+import { useNetwork } from 'store/network';
+import { usePointCursor } from 'store/pointCursor';
+import { usePointCache } from 'store/pointCache';
+import { useHistory } from 'store/history';
+
+import * as need from 'lib/need';
+import useCurrentPointName from 'lib/useCurrentPointName';
+import useEthereumTransaction from 'lib/useEthereumTransaction';
+import { GAS_LIMITS } from 'lib/constants';
+import { ETH_ZERO_ADDR } from 'lib/wallet';
+import useLifecycle from 'lib/useLifecycle';
+
+import ViewHeader from 'components/ViewHeader';
+import MiniBackButton from 'components/MiniBackButton';
+import InlineEthereumTransaction from 'components/InlineEthereumTransaction';
+import View from 'components/View';
+
+function useRejectTransfer() {
+  const { contracts } = useNetwork();
+  const { pointCursor } = usePointCursor();
+  const { syncDetails, syncControlledPoints } = usePointCache();
+
+  const _contracts = need.contracts(contracts);
+  const _point = need.point(pointCursor);
+
+  const { construct, ...rest } = useEthereumTransaction(
+    useCallback(
+      () =>
+        azimuth.ecliptic.setTransferProxy(_contracts, _point, ETH_ZERO_ADDR),
```

Comment:
> I get a revert when attempting to set the transfer proxy to 0 (as the current transfer proxy)

Right. Azimuth will eventually want to support this, for this and similar use cases, but doesn't allow proxies setting proxies right now. Locally hiding is fine, looks like it's time to build that better `/lib/cookie`...

Alternatively, we could transfer it to the current owner, with the `reset` arg as `true`, but I'm not sure if that's more desirable.

Reply:
@Fang- can you ask Alex/Jimmy in today's meeting? I won't make it. I'll either import or write a simple local storage hook and that'll work just fine.

Reply:
Oh snap, I forgot to ask for input on this. Sorry! But local storage seems sane and good.